README.md

```diff
@@ -1,5 +1,27 @@
-# dcs-for-azure-templates
-A collection of templates that can be used to leverage Delphix Compliance Services in Azure.
+## Purpose
+Azure Data Factory supports 100+ connectors, including those for SaaS applications and PaaS databases. Data that resides
+in disparate locations often needs masking in order to be leveraged by engineers for feature development or issue
+triage and debugging. These things must never be done in production as the risk to data exposure is too great. Data
+protection is alo required in accordance with regulations such as GDPR, FINRA, CCPA, HIPAA, and PCI DSS. For these
+reasons, it is essential that it's possible to identify and mask sensitive data from production into lower environments.
+
+Delphix provides a capability to leverage Microsoft's Azure Data Factory in order to perform automated sensitive data
+discovery, assign masking algorithms, and mask sensitive data such that it maintains referential integrity.
+
+## What Is This Code For?
+
+This repository contains a collection of templates that can be used to leverage Delphix Compliance Services in Azure
+for these purposes.
+
+To leverage these templates you will need a Delphix Compliance Services account, for that more information can be found
+here: https://www.perforce.com/products/delphix/compliance-services
+
+Don't see a data source you're hoping to profile and mask? If you feel compelled, you can use these provided templates
+as a jumping off point and create a new template to discover and mask sensitive data using the Delphix compliance
+services APIs. Such [contributions](./README.md#Contribution) are more than welcome and if you would like engineering
```

Comment:
Feel free to replace or drop this suggestion. 

```
These contributions are highly appreciated, and if you need engineering support tailored to your specific use case, they will enable our engineering team to assist you in case any issues arise.
```